source/vibe/http/websockets.d

```diff
@@ -697,108 +727,234 @@ final class IncomingWebSocketMessage : InputStream {
 	}
 }
 
+/// Magic string defined by the RFC for challenging the server during upgrade
+private static immutable s_webSocketGuid = "258EAFA5-E914-47DA-95CA-C5AB0DC85B11";
```

Comment:
Actually at module level `immutable x = y` still have the same behavior. Do you want me to revert that change ?